File: app/src/main/java/com/rinconadadelsur/sistemas/bioseguridad/Entidades/eRegistroManto.java
package com.rinconadadelsur.sistemas.bioseguridad.Entidades;

public class eRegistroManto {

    private String _codigo;
    private String _elemento;
    private String _descripcion;

    public eRegistroManto(){

    }

    public eRegistroManto(String _codigo, String _elemento, String _descripcion) {
        this._codigo = _codigo;
        this._elemento= _elemento;
        this._descripcion = _descripcion;
    }

    public String get_codigo() {
        return _codigo;
    }

    public void set_codigo(String _codigo) {
        this._codigo = _codigo;
    }

    public String get_elemento() {
        return _elemento;
    }

    public void set_elemento(String _elemento) {
        this._elemento = _elemento;
    }

    public String get_descripcion() {
        return _descripcion;
    }

    public void set_descripcion(String _descripcion) {
        this._descripcion = _descripcion;
    }

    @Override
    public String toString() {
        return _descripcion;
    }
}

File: app/src/main/java/com/rinconadadelsur/sistemas/bioseguridad/Entidades/eRegistroManto.kt
package com.rinconadadelsur.sistemas.bioseguridad.Entidades

class eRegistroManto {
    private var _codigo: String? = null
    private var _elemento: String? = null
    private var _descripcion: String? = null

    constructor()

    constructor(_codigo: String?, _elemento: String?, _descripcion: String) {
        this._codigo = _codigo
        this._elemento = _elemento
        this._descripcion = _descripcion
    }

    fun get_codigo(): String? {
        return _codigo
    }

    fun set_codigo(_codigo: String?) {
        this._codigo = _codigo
    }

    fun get_elemento(): String? {
        return _elemento
    }

    fun set_elemento(_elemento: String?) {
        this._elemento = _elemento
    }

    fun get_descripcion(): String {
        return _descripcion!!
    }

    fun set_descripcion(_descripcion: String) {
        this._descripcion = _descripcion
    }

    override fun toString(): String {
        return _descripcion!!
    }
}
